### PR TITLE
Allow periods in qube names

### PR DESCRIPTION
**Pull request by DemiMarie**
This allows periods in qube names by using the function from libqubes-pure to validate them instead of the built-in version.

Part of QubesOS/qubes-issues#8199.

### CODE REVIEW

**Comment by marmarek** [comment]
PipelineRetry

File: debian/control [hunk 1]
```diff
@@ -19,7 +19,8 @@ Build-Depends:
  libpng-dev,
  libnotify-dev,
  qubesdb-dev,
- help2man
+ help2man,
+ qubes-utils-dev (>= 4.2.10)
```
Comment (marmarek) [inline debian/control:23]: There is no such package. It seems you put it into `libqubes-rpc-filecopy-dev`.

**Review by marmarek — CHANGES_REQUESTED** [review]
This fails to build due to not linking with libqubes-pure.

File: debian/control (hunk 1 above)
Comment (DemiMarie) [reply debian/control:23]: Should I change that?  `libqubes-rpc-filecopy-dev` is obviously wrong.

Comment (marmarek) [reply debian/control:23]: The proper Debian way would be a new subpackage. I think we don't link against that library anywhere yet (this is the first PR doing it, so it's still ok to fix packaging there.

Comment (DemiMarie) [reply debian/control:23]: I’m pretty sure that qfile-unpacker links to libqubes-pure already.

Comment (marmarek) [reply debian/control:23]: But fortunately, not directly, but through a library that is also in linux-utils, so splitting the package should still be safe-ish. If you want, I can do it, and try to setup dependencies properly.

Comment (DemiMarie) [reply debian/control:23]: That would be great, thanks.

Comment (marmarek) [reply debian/control:23]: Package is split, you can adjust dependency now.

**Review by marmarek — APPROVED** [review]

**Pull request merged**